internal/blockchain/v0/reactor.go

```diff
@@ -139,6 +141,7 @@ func NewReactor(
 		peerUpdatesCh: make(chan p2p.Envelope),
 		closeCh:       make(chan struct{}),
 		metrics:       metrics,
+		syncStartTime: time.Now(),
```

Comment:
does this mean that in nodes that haven't done state sync tendermint reports the interval as "time since app started" would this be confusing to users?

Reply:
This is fast sync. Not state sync.

Reply:
Sorry, about the confusion, I think the question holds: if the node starts but never fastsyncs, (either because we're statesyncing, waiting for genesis, or starting conensus,) then you get "uptime" for something where that means "never" which isn't great.

Reply:
Yeah I'm not really sure we need to worry about that case. If fast sync never happens, then this field can be ignored. Alternatively, we can just use zero time if fast sync never starts.

But I don't really think it matters tbh.

Reply:
@tychoish I got what you mean. will assign 0 to it and then update it when the fastsync starts.